torchci/rockset/commons/__sql/weekly_force_merge_stats.sql

```diff
@@ -0,0 +1,174 @@
+-- gets percentage of total force merges, force merges with failures, and force merges without failures (impatient)
+WITH
+    issue_comments AS(
```

Comment:
Just curious what is `issue_comments` is used for here.  I guess it's for get rid of bot accounts?

Reply:
Its to get a more accurate time

Reply:
Ah, that makes sense.  The time of the merge record on Rockset isn't when merging is done, not the time of the comment.  For force merges, the 2 values would be pretty close I think